kahuna/public/js/image/view.html

```diff
@@ -40,13 +40,13 @@
                                 <a draggable="true"
                                     ui:sref="{ crop: null }">
                                     <img class="image-crop__image" ng:src="{{ ctrl.image.data.thumbnail | assetFile }}" />
-                                </a>
                                 <div class="image-crop__aspect-ratio"
                                     ng:class="{'image-crop__aspect-ratio--selected': !ctrl.crop}">
                                     <ul>
                                         <li>{{ctrl.image.data.source.dimensions.width}} &times; {{ctrl.image.data.source.dimensions.height}}</li>
                                     </ul>
                                 </div>
```

Comment:
Can we please adjust the indentation of all the elements now enclosed by the `a` link? I've recently fixed a bug in Composer that was mostly due to markup being hard to track because of broken indentation :-)